WordPress/WordPressUITests/Utils/XCTest+Extensions.swift

```diff
@@ -77,4 +77,21 @@ extension XCTestCase {
         static let category = "iOS Test"
         static let tag = "tag \(Date().toString())"
     }
+
+    public func removeApp() {
+        let app = XCUIApplication()
+        app.terminate()
```

Comment:
Not actually related to this line, but I noticed that `app.terminate()` is also being called [in setUp](https://github.com/wordpress-mobile/WordPress-iOS/blob/571370616c8d007f2b6b5825248ab937919ae556/WordPress/WordPressUITests/Utils/XCTest%2BExtensions.swift#L13). Since we're removing the app after every test, do we still need that in place during setUp?

Reply:
No, we don't. Thanks for pointing that out! 🙇‍♂️ 
Removed it in [83ddaa7](https://github.com/wordpress-mobile/WordPress-iOS/pull/19026/commits/83ddaa7ff228f412b630f345a2b3e13a4953a17e).